randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -813,12 +825,14 @@ Extra - asset_id: collision_camera_045
   * Extra - actor_name: LE_Platform_AmmoCharge_002
   * Extra - actor_type: weightactivatedplatform
   > Pickup (Power Bomb Tank)
-      Any of the following:
-          Spider Boost (Beginner) and Can Spider Boost
-          All of the following:
-              Infinite Bomb Jump (Advanced) and Lay Bomb
-              # Defeat the Gulluggs
-              Combat (Advanced) or Defeat Weak Enemies
+      All of the following:
+          Disabled Final Boss Diggernaut
```

Comment:
why can you only collect the item if diggernaut is not the final boss?

Reply:
The game ends after beating Diggernaut, so you never get to collect the item.

Reply:
hmm, but theoretically if you would start behind diggernaut then you would be able to collect the item and beat diggernaut, no?

Reply:
It's a bit more complicated then that. The item is "accessible" pre-fight, but requires OoB stuff. I probably should disable early access period.

Reply:
In that theoretical case, would you be able to backdoor diggernaut?

Reply:
The fight currently only starts if you enter from the right door. If you aren't OoB, you can't backdoor because of the wall in the arena.